samples/document-security/README.md

```diff
@@ -23,62 +23,66 @@ The [azure-search-openai-demo](/) project can set up a full RAG chat app on Azur
 ## Table of Contents
 
 - [Requirements](#requirements)
-- [Setting up Microsoft Entra ID Apps](#setting-up-entra-id-apps)
+- [Setting up Microsoft Entra applications](#setting-up-microsoft-entra-applications)
```

Comment:
This file is just a copy paste of login_and_acl.md

Reply:
can we eventually remove this? OK to leave it for now

Reply:
I assume you mean, can we we eventually remove the duplicate copy of login_and_acl.md?

Arun wants the ACL feature to show up as a completely separate sample in the Microsoft Learn Samples browser, which requires the copy. If it's annoying to maintain, we could discuss removing it, and it wouldn't show up in the samples browser anymore.